src/main/java/net/devtech/arrp/mixin/LifecycledResourceManagerImplMixin.java

```diff
@@ -39,6 +39,8 @@ private static List<ResourcePack> registerARRPs(List<ResourcePack> packs, Resour
 		if (optionalInt.isPresent()) {
 			ARRP_LOGGER.info("ARRP register - between vanilla and mods");
 			SidedRRPCallback.BETWEEN_VANILLA_AND_MODS.invoker().insert(type, copy.subList(0, optionalInt.getAsInt()));
+			ARRP_LOGGER.info("ARRP register - between mods and user");
+			SidedRRPCallback.BETWEEN_MODS_AND_USER.invoker().insert(type, copy.subList(0, optionalInt.getAsInt()+1));
```

Comment:
shouldn't the index be adjusted for how many between_vanilla_and_mod resource packs get added?

Reply:
Oh shoot, you're right!